crates/web-sys/webidls/enabled/RTCRtpSender.webidl

```diff
@@ -51,28 +51,26 @@ dictionary RTCRtcpParameters {
   boolean   reducedSize;
 };
 
-dictionary RTCRtpCodecParameters {
-  unsigned short payloadType;
-  DOMString      mimeType;
-  unsigned long  clockRate;
-  unsigned short channels = 1;
-  DOMString      sdpFmtpLine;
+dictionary RTCRtpCodec {
+  required DOMString     mimeType;
+  required unsigned long clockRate;
+  unsigned short         channels;
+  DOMString              sdpFmtpLine;
 };
 
+dictionary RTCRtpCodecParameters : RTCRtpCodec {
+  required octet payloadType;
```

Comment:
```suggestion
  required unsigned short payloadType;
```
This would be a breaking change otherwise.

Reply:
I think `required` also has to be removed, because the `new()` function has changed now.